### ISSUE TITLE
New records of 2nd level models should assume parent = 'project'

### ISSUE THREAD

**Issue by dtm2451** [mountetna/monoetna#292]
I can still create orphan records at the level of models that hang off of the project. Ideally, such records could have an assumed parent and thus never be orphans.

**Comment by coleshaw** [comment]
Some projects (IPI) use disconnected records to hide them for a purpose, so it is possible to disconnect records at any level...

I think at some point we've talked about creation of new records must be attached to the graph, but you can disconnect later once the record exists. But that hasn't been implemented.

**Comment by dtm2451** [comment]
Right, this issue is specifically about records of models that attach directly to 'project' as I think there should be an implicit parent for any new record.

But hmm is it actually possible to remove a parent attribute after it's already there?  Might need to create that pathway first for the cases you mention!

**Comment by coleshaw** [comment]
It is possible via the API and Timur to remove parent attributes from records.

**Comment by coleshaw** [comment]
Well, like blank or null out the parent attribute value, not like remove it from the model...
